Lzyct/flutter_native_splash
# feat: support adjust padding bottom for branding image

## What it does

- Support adding bottom padding for branding image

## How to test

1. follow this documentation to run and generate the native splash screen
2. enable branding image
3. add this config on your config yaml
```yaml
   branding_bottom_padding: 24

  #branding_bottom_padding_ios: 24 #specific config for iOS
  #branding_bottom_padding_android: 24 #specific config for Android
```
5. observe the branding image position

## Screenshot

#### iPhone 15 Pro Max 
![image](https://github.com/Lzyct/flutter_native_splash/assets/1531684/49ef6d58-8e45-49ac-9747-eb0a42e74a46)


### iPhone SE
![image](https://github.com/Lzyct/flutter_native_splash/assets/1531684/bc8238b2-c9e2-471d-8834-494fa049448d)

### < Android 13
![image](https://github.com/jonbhanson/flutter_native_splash/assets/1531684/d17b062e-633a-4787-b06d-b2c3b6835a28)
